### ISSUE TITLE
Rename precisions e.g. P10000 -> E5

### ISSUE THREAD

**Issue by hdgarrood** [lumihq/purescript-fixed-precision#23]
This way, this library is more mutually intelligible with the corresponding Haskell library, and also I think the exponential notation is much easier to read and understand quickly. We can export type synonyms under the old names for backwards compatibility.

**Comment by hdgarrood** [comment]
cc @arthurxavierx @codedmart thoughts?

**Comment by arthurxavierx** [comment]
Yes, I think this is an excellent idea. The current naming scheme makes things a bit noisy when using some rather high precisions.
